WalletWasabi.Fluent/Styles/TextBox.axaml

```diff
@@ -123,10 +123,14 @@
     <Setter Property="IsVisible" Value="True" />
   </Style>
 
-  <Style Selector="TextBox.selectable_text_block">
+  <Style Selector="TextBox.selectableTextBlock">
     <Setter Property="IsReadOnly" Value="True" />
     <Setter Property="Margin" Value="0" />
     <Setter Property="Padding" Value="0" />
+    <Setter Property="Background" Value="Transparent" />
+    <Setter Property="BorderThickness" Value="0" />
+    <Setter Property="MinHeight" Value="0" />
+    <Setter Property="MinWidth" Value="0" />
```

Comment:
What do you think about setting the CaretBrush to `Transparent` here?

Reply:
I think because its readonly it doesnt show? but added anyway.